src/scripts/main.js

```diff
@@ -1,3 +1,23 @@
 'use strict';
 
-// write code here
+const inputs = document.querySelectorAll('input');
+const parent = document.querySelectorAll('.field');
+
+inputs.forEach((item, index) => {
+  const label = document.createElement('label');
+
+  label.classList.add('field-label');
+  label.htmlFor = item.id;
+  label.textContent = item.getAttribute('name');
+
+  parent[index].insertBefore(label, item);
+
+  let placeholder = item.getAttribute('name').split('');
+
+  placeholder[0] = placeholder[0].toUpperCase();
+  placeholder = placeholder.join('');
```

Comment:
```suggestion
  placeholder = item.name[0].toUpperCase() + item.name.slice(1);
```
Может просто сделать вот так, и не лохматить бабушку?